sdk/communication/azure-communication-identity/swagger/README.md

```diff
@@ -0,0 +1,57 @@
+# Azure Communication Identity library for Java
+
+> see https://aka.ms/autorest
+## Getting Started
+
+To build the SDK for Communication Identity library, simply Install AutoRest and in this folder, run:
+
+### Setup
+```ps
+Fork and clone https://github.com/Azure/autorest.java
+git checkout v4
+git submodule update --init --recursive
+mvn package -Dlocal
+npm install
+npm install -g autorest
+```
+
+### Generation
+
+```ps
+cd <swagger-folder>
+autorest README.md --java --v4 --use=@autorest/java@4.0.1 --tag=identity
+```
+
+### Tag: identity
+
+These settings apply only when `--tag=identity` is specified on the command line.
+
+``` yaml $(tag) == 'identity'
+input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/14bfbf5d0ff8f0dc1358e6e60362e99d0a649ba7/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json
```

Comment:
Let's use the final commit that got merged yesterday, 5b19c6e69cd2bb9dbe4e5c1237b2c5a175d90ca5

Reply:
(Y) Sounds good, using the master version: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/communication/data-plane/Microsoft.CommunicationServicesIdentity/stable/2021-03-07/CommunicationIdentity.json